packages/playout-gateway/src/tsrHandlerJobs/jobQueue.ts

```diff
@@ -0,0 +1,231 @@
+import { EventEmitter } from 'stream'
+import { AbortError } from 'timeline-state-resolver'
+import { Logger } from 'winston'
+import { Job } from './job'
+
+interface JobDescription<X> {
+	id?: string
+	job: Job<X, any, any>
```

Comment:
Propably nothing/not possibly, but is there really no other type than `any`, that can be applied?